xml/slert_systemd_shielding.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+
+<chapter version="5.0" xml:id="cha-shielding-with-systemd"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>Shielding with systemd</title>
+ <info/>
+ <para>
+  systemd has native support for the cpuset controller since version 244.
+  Shielding the sensitive workload can be achieved with the proper configuration of respective units.
+  This is only supported with cgroup unified hierarchy (v2) and hence the shielded
+  vs. unshielded division copies the structure of typical systemd cgroup tree.
+ </para>
+
+ <sect1 xml:id="sec-systemd-shielding-teardown">
+   <title>Setup of the shield</title>
+   <para>
+        The general idea is to have one cpuset for the main sensitive workload
+        and a complementary cpuset for the supporting tasks.
+        Resources are distributed in the top-down fashion, so to ensure proper
+        allocation for the main workload we must take into consideration all the
+        top-level cgroups on the system.
+        systemd by default creates the following units:
+           <literal>init.scope</literal>,
+           <literal>system.slice</literal>,
+           <literal>user.slice</literal> and
+           <literal>machine.slice</literal>.
+   </para>
+   <para>
+           We must configure <emphasis>all</emphasis> of these units not to stand in the way of our main workload.
+           For instance with following
+	   <link
+	   xlink:href="https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-systemd.html#sec-boot-systemd-custom-drop-in">
```

Comment:
Is it still 15-SP**3**? Should this be adapted to 15-SP**4**? If you remove the "-SP3" you can have the "latest" version.

Reply:
I'm slightly worried the "latest" might break in the future but it captures the meaning. Hopefully, users would have enough search terms from the URL itself if that happened.